src/lib.rs

```diff
@@ -88,42 +90,167 @@ impl InlinerBuilder {
         };
         let result =
             Visitor::<R>::with_resolver(src_file, self.root, errors.as_mut(), Cow::Owned(resolver))
-                .visit();
+                .visit()
+                .map_err(|kind| Error::Initial(kind))?;
```

Comment:
```suggestion
                .map_err(Error::Initial)?;
```

Minor change to keep clippy happy.